### PR TITLE
Various Fixes for Gnome Extension

### PR DESCRIPTION
**Pull request by soumyaDghosh**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `make lint`?
- [ ] Have you successfully run `pytest tests/unit`?

-----
## Changes:
- The gnome-sdk has webkit2gtk-4.1 files, but it also needs a layout just like webkit2gkt-4.0. 
- Changed the `SNAPCRAFT` in the layouts to `CRAFT`
- Updated description[Gnome SDK now has Gnome-45 libraries also]

### CODE REVIEW

File: snapcraft/extensions/gnome.py [hunk 1]
```diff
@@ -147,6 +147,7 @@ def get_root_snippet(self) -> Dict[str, Any]:
             "environment": {
                 "SNAP_DESKTOP_RUNTIME": "$SNAP/gnome-platform",
                 "GTK_USE_PORTAL": "1",
+                "GDK_DEBUG": "portals",
```
Comment (kenvandine) [inline snapcraft/extensions/gnome.py:150]: I'd separate this as it's unrelated to the other changes in the PR.  Also, I'm not confident of the implications we'll have across the wide range of snaps we have using the extension, so I'd say it's a pretty risky change.

**Review by sergio-costas — APPROVED** [review]
LGTM, although it seems that the linter has some complains...

**Review by sergiusens — CHANGES_REQUESTED** [review]
you have not changed any unit tests, I would make 2 PRs, one for the text change and another for the layout changes

**Comment by soumyaDghosh** [comment]
> you have not changed any unit tests, I would make 2 PRs, one for the text change and another for the layout changes

Should I close this PR then?

**Comment by sergiusens** [comment]
> > you have not changed any unit tests, I would make 2 PRs, one for the text change and another for the layout changes
> 
> Should I close this PR then?

what I mean is that this PR is failing as you changed code but not their related unit tests

**Comment by sergiusens** [comment]
if you want to close and create two new from scratch PRs go ahead, or trim this down to one thing (the text change perhaps)

**Comment by soumyaDghosh** [comment]
recreating this PR, with some added changes

**Pull request closed without merging**